parts/k8s/addons/secrets-store-csi-driver.yaml

```diff
@@ -282,6 +282,20 @@ status:
   conditions: []
   storedVersions: []
 ---
+{{- /* A priority class for the daemonset such that they are not */}}
+{{- /* frozen out of a node due to the node filling up with "normal" */}}
+{{- /* pods before the daemonset controller can get the daemonset */}}
+{{- /* pods to be scheduled. */}}
+apiVersion: scheduling.k8s.io/v1
```

Comment:
cc @aramase @chewong this change was made due to observed scheduling contention on clusters at scale (i.e., nodes coming on line but filling up so quickly that csi-secrets-store pods were not able to be scheduled via daemonset

Should we make such a change to the upstream reference specs as well?